spec/javascript/e2e/LeaseUpApplicationsPage/status_update.e2e.js

```diff
@@ -0,0 +1,119 @@
+
+        // Fill out the status modal and submit.
+        await fillOutAndSubmitStatusModal(page)
+
+        // Wait for the api call
+        await checkForStatusUpdateSuccess(page, FIRST_ROW_LEASE_UP_APP_ID)
+
+        // Get changed status, it should be different
+        const currentStatus = await sharedSteps.getText(page, firstRowStatusDropdown)
+        expect(currentStatus).not.toBe(originalStatus)
+      },
+      DEFAULT_E2E_TIME_OUT
+    )
+  })
+  describe('using the bulk update checkboxes', () => {
+    const nthRowStatusDropdown = (n) =>
+      `.rt-tr-group:nth-child(${n}) .rt-td .dropdown .dropdown-button`
+    const bulkStatusDropdown = '.filter-row .status-dropdown__control .dropdown-button'
+
+    const checkboxById = (appId) => `#bulk-action-checkbox-${appId}`
+    const statusMenuItem = (value) => `li.is-${value.toLowerCase()} a`
+
+    const PROCESSING = 'Processing'
+    const APPEALED = 'Appealed'
+    test(
+      'should change the status for selected checkboxes',
+      async () => {
+        const { page } = await SetupBrowserAndPage(testBrowser, true)
+        await waitForLeaseUpAppTableToLoad(page)
+
+        // Check the checkboxes in the 2nd and 3rd row
+        await page.click(checkboxById(SECOND_ROW_LEASE_UP_APP_ID))
+        await page.click(checkboxById(THIRD_ROW_LEASE_UP_APP_ID))
+        expect(
+          await sharedSteps.getCheckboxVal(page, checkboxById(SECOND_ROW_LEASE_UP_APP_ID))
+        ).toBe(true)
+        expect(
+          await sharedSteps.getCheckboxVal(page, checkboxById(THIRD_ROW_LEASE_UP_APP_ID))
+        ).toBe(true)
+
+        // Select Processing as bulk status
+        await selectStatusDropdownValue(page, bulkStatusDropdown, statusMenuItem(PROCESSING))
+        // Fill out the status modal and submit
+        await fillOutAndSubmitStatusModal(page)
+
+        // Wait for field update comment requests to complete and fail if they were unsuccessful.
+        // The problem is that if one of these requests does not happen we get an ambiguous timeout error.
+        await checkForStatusUpdateSuccess(page, SECOND_ROW_LEASE_UP_APP_ID)
+        await checkForStatusUpdateSuccess(page, THIRD_ROW_LEASE_UP_APP_ID)
+        // Expect checkboxes to be unchecked and statuses to be updated
+        let secondRowUpdatedStatus = await sharedSteps.getText(page, nthRowStatusDropdown(2))
+        expect(secondRowUpdatedStatus).toBe(PROCESSING)
+        let thirdRowUpdatedStatus = await sharedSteps.getText(page, nthRowStatusDropdown(3))
+        expect(thirdRowUpdatedStatus).toBe(PROCESSING)
+        expect(
+          await sharedSteps.getCheckboxVal(page, checkboxById(SECOND_ROW_LEASE_UP_APP_ID))
+        ).toBe(false)
+        expect(
+          await sharedSteps.getCheckboxVal(page, checkboxById(THIRD_ROW_LEASE_UP_APP_ID))
+        ).toBe(false)
+
+        // Repeat with status Appealed
+        await page.click(checkboxById(SECOND_ROW_LEASE_UP_APP_ID))
```

Comment:
this ends up feeling very repetitive but is maybe worth it to make sure that we're always actually changing the status

Reply:
yeah I think it's okay